### PR TITLE
tests written

### PR DESCRIPTION
**Pull request by easyra**

### CODE REVIEW

File: basic-javascript/project-1.spec.js [hunk 1]
```diff
@@ -1,3 +1,130 @@
...
+})
+
+it('should equal -3', () => {
+    const expected = 2
+    const actual = helpers.subtractFive(expected)
+
+    expect(actual).toEqual(-3)
+})
+
+it('should equal same length', () => {
```
Comment (bradencodes) [inline basic-javascript/project-1.spec.js:20]: Change the wording to say what the test should return and why. Something like, `it('should return true when two strings have the same length')`

File: basic-javascript/project-1.spec.js [hunk 2]
```diff
@@ -1,3 +1,130 @@
...
+    const actual = helpers.areSameLength('rainbow', 'seventy')
+
+    expect(actual).toEqual(true)
+})
+
+it('should equal true', () => {
+    const actual = helpers.areEqual(2,2)
+    expect(actual).toEqual(true)
+})
```
Comment (bradencodes) [inline basic-javascript/project-1.spec.js:29]: Have a test to also make sure the function returns false when the inputs are not equal.

File: basic-javascript/project-2.spec.js [hunk 3]
```diff
@@ -1,3 +1,146 @@
...
+it('should equal 20', () => {
+    const actual = funcs.getBiggest(20,10)
+
+    expect(actual).toEqual(20)
+})
+
+it('should equal Guten Tag!', () => {
+    const actual = funcs.greeting('German')
+
+    expect(actual).toEqual("Guten Tag!")
```
Comment (bradencodes) [inline basic-javascript/project-2.spec.js:17]: Write tests for the other languages as well.

File: basic-javascript/project-2.spec.js [hunk 4]
```diff
@@ -1,3 +1,146 @@
...
+it('should equal true', () => {
+    const actual = funcs.isTenOrFive(10)
+
+    expect(actual).toEqual(true)
+})
+
+it('should equal true', () => {
+    const actual = funcs.isInRange(21)
+
+    expect(actual).toEqual(true)
```
Comment (bradencodes) [inline basic-javascript/project-2.spec.js:29]: Test for what happens when you don't input a number into the function.

**Comment by bradencodes** [comment]
## MVP
- [x] Write tests for project-1.js
- [x] Write tests for project-2.js

## Stretch
- [x] Correct bugs and refactor the code
- [ ] Write tests for arrays.js
- [ ] Write tests for object.js

## What Looks Great
- MVP complete! Nice work
- You corrected some bugs and started writing tests for arrays.js
- Code is well formatted and easy to read
- Titles for tests are descriptive, but could be tweaked a little

## What Could Be Improved
- MVP needs to be completed
- Need more commits. You should have one commit for every one change.
- Commits need more description. Check out this guide (https://chris.beams.io/posts/git-commit/)
- Tests need to be more thorough and check for edge cases. Also check that functions fail as expected, such as returning false when two numbers are not equal or null when the input data is not the correct type.

## What Could Be Added
- Add a `describe` for each function, and have an `it` for every test about that function.

## Rating (1-3)
**3**
